extension-manifest-v3/src/background/adblocker.js

```diff
@@ -246,16 +246,22 @@ ${scripts.join('\n\n')}}
   // wrapper to break the "isolated world" so that the patching operates
   // on the website, not on the content script's isolated environment.
   function codeRunningInContentScript(code, nonce, nonceSize) {
+    let trustedTypePolicy;
     if (window.trustedTypes && window.trustedTypes.createPolicy) {
-      window.trustedTypes.createPolicy('default', {
+      trustedTypePolicy = window.trustedTypes.createPolicy(nonce, {
```

Comment:
I would avoid using the nonce here. If there is an API to list them, for instance. But I think you can use any name as long as it is likely to avoid collisions.